### DISCUSSION TITLE
(Newbie question) Is there a way to change the color of this bar?

### DISCUSSION THREAD

**Discussion by morenohernan** [doublecmd/doublecmd#1270] [General]
Hi, 
I just discovered doublecmd and I like it a lot. I've customized it according to my preferences. I'd like to change the color of this bar, I cannot find where.. is it possible?

<img width="428" alt="image" src="https://github.com/doublecmd/doublecmd/assets/700619/da35557c-1161-4fe7-b319-bfcc0b64c1cd">

Thanks in advance..

**Comment by alexx2000** [comment]
Hi,
Yes, it is possible. What DC version do you use?

**Reply by morenohernan** [reply]
Hi, 
This is the version

<img width="151" alt="image" src="https://github.com/doublecmd/doublecmd/assets/700619/c5a982d1-29f0-440a-8c6a-57139bdd5424">

**Comment by alexx2000** [comment]
https://skif-off.github.io/doublecmd.github.io/doc/en/configuration.html#ConfigColor

**Reply by morenohernan** [reply]
Thanks a lot.
